corehq/apps/app_manager/xform.py

```diff
@@ -948,6 +948,17 @@ def get_questions(self, langs, include_triggers=False,
                     "type": "DataBindOnly",
                     "calculate": bind.attrib.get('calculate') if hasattr(bind, 'attrib') else None,
                 }
+
+                # Include meta information about the stock entry
+                if data_node.tag_name == 'entry':
+                    parent = next(data_node.xml.iterancestors())
+                    if parent:
```

Comment:
this is fairly fragile and i'm planning on making more robust by ensuring the parent tag_name is one of balance, transfer, dispense, or receive.